### DISCUSSION TITLE
v2.7.5

### DISCUSSION THREAD

**Discussion by NayamAmarshe** [upscayl/upscayl#437] [Releases]
# Upscayl v2.7.5 🥳 
## What's Changed 🤓
* New website launch! 🎉
* Fixed 2x, 3x scale issue #411 😌
* Added compression and fixed JPEG/PNG output forever 😤
* Added output type and compression options for batch upscale 💪🏻
* Added fix for empty upscaled image #426 🫙
* Added a totally new Upscayl theme. 💅🏻
* Fix scale resolution #433 ⚖️
* Started to integrate Upscayl Cloud (more at https://upscayl.org) ☁️
* Added toggle to allow overwriting upscaled photos. 🚨
* Added new macOS look. 🥰
* Batch upscale will now delete .DS_Store files in macOS. 🗑️

## Contributors 🤩
* @udu3324 made their first contribution in https://github.com/upscayl/upscayl/pull/355

Full Changelog: https://github.com/upscayl/upscayl/compare/v2.5.5...v2.7.5

<hr /><em>This discussion was created from the release <a href='https://github.com/upscayl/upscayl/releases/tag/v2.7.5'>v2.7.5</a>.</em>

**Comment by aaronliu0130** [comment]
@NayamAmarshe Shouldn't the issues linked be #303 and #354 instead of #411 and #426?

**Comment by NayamAmarshe** [comment]
@aaronliu0130 I just linked the first issue that I could find related to it 😅

**Comment by wywhynot** [comment]
The before/after feature is broken now. All results end up looking like this
![bandicam 2023-09-04 00-00-59-593](https://github.com/upscayl/upscayl/assets/48932550/86460990-52b4-45ed-98b4-6f9dd67ff770)
![bandicam 2023-09-04 00-00-49-173](https://github.com/upscayl/upscayl/assets/48932550/5161d360-848a-44ee-b55d-23fb71c6abf1)

**Reply by aaronliu0130** [reply]
@wywhynot This seems to be an instance of #354 which should've been fixed in this update. The output image's also gone, right? If so can you also post the logs in #354?

**Comment by wywhynot** [comment]
let me say that i've found out if I click upscayl a 2nd time, the
comparison view appears as it should. How do I post a log?

**Reply by aaronliu0130** [reply]
Well logs when it doesn't happen aren't useful and according to our recent research in #354 it's not an issue that can be logged, rather something funky happening with a dependency, so we don't really need the logs anymore. If you still want to know how it's at the top of settings.

**Reply by NayamAmarshe** [reply]
> let me say that i've found out if I click upscayl a 2nd time, the
> comparison view appears as it should. How do I post a log?
> 
> On Mon, Sep 4, 2023 at 9:42 AM Aaron Liu ***@***.***> wrote:
> 
> > @wywhynot <https://github.com/wywhynot> This seems to be an instance of
> > #354 <https://github.com/upscayl/upscayl/issues/354> which should've been
> > fixed in this update. The output image's also gone, right? If so can you
> > also post the logs in #354 <https://github.com/upscayl/upscayl/issues/354>
> > ?
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/orgs/upscayl/discussions/437#discussioncomment-6904998>,
> > or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/ALVKNRSHFHYWDXBLM3LFHQLXYXLE5ANCNFSM6AAAAAA4HWNDW4>
> > .
> > You are receiving this because you were mentioned.Message ID:
> > ***@***.***>
> >
> 

Yes, that happens because when you click Upscayl again, it refetches the upscaled image and renders it.

It's possible that images that are freshly upscaled need some time to save before they're shown in the viewer. We're thinking of adding a 500ms delay between saving the image and previewing it.

**Comment by shanyingji** [comment]
新版本在运行的时候软件会崩溃，疑似爆内存，但是老版本就不会出现这个问题，请问如何下载2.5.5的老版本呢？

**Reply by shanyingji** [reply]
⚙️ Getting saveImageAs from localStorage jpg
⚙️ Getting model from localStorage {"label":"General Photo (Real-ESRGAN)","value":"realesrgan-x4plus"}
⚙️ Setting gpuId to empty string
🔀 Setting model to 
🎯 GET_MODELS_LIST:  
🔄 Resetting image paths
🔀 Setting model to realesrgan-x4plus
⚙️ Getting saveImageAs from localStorage jpg
⚙️ Getting model from localStorage {"label":"General Photo (Real-ESRGAN)","value":"realesrgan-x4plus"}
⚙️ Setting gpuId to empty string
⚙️ Getting rememberOutputFolder from localStorage false
⚙️ Getting saveImageAs from localStorage jpg
⚙️ Getting model from localStorage {"label":"General Photo (Real-ESRGAN)","value":"realesrgan-x4plus"}
⚙️ Setting gpuId to empty string
🔀 Setting model to 
🔀 Setting model to realesrgan-x4plus
⚙️ Getting saveImageAs from localStorage jpg
⚙️ Getting model from localStorage {"label":"General Photo (Real-ESRGAN)","value":"realesrgan-x4plus"}
⚙️ Setting gpuId to empty string
⚙️ Getting rememberOutputFolder from localStorage false

**Reply by aaronliu0130** [reply]
已知问题，见#439。我们加了个有点问题的照片压缩。https://github.com/upscayl/upscayl/releases

**Comment by IkelAtomig** [comment]
You didn't update the Flatpak release.

**Reply by NayamAmarshe** [reply]
Yes, because 2.7.5 has a few bugs and 2.8 is just releasing very soon.
